### DISCUSSION TITLE
Sustainability thread

### DISCUSSION THREAD

**Discussion by erjank** [cssi-pi-community/2022-meeting#75] [Open discussion 1 (Monday afternoon)]
Examples of projects that have 10-year track records?

**Comment by klehnert55** [comment]
IGSN (International Generic Samples Number, https://www.igsn.org): Persistent Identifier system for physical samples, developed 2004-2009 with NSF funding, that became an International non-profit organization IGSN e.V. in 2011. In 2019-2021 the IGSN e.V. was funded by the Sloan Foundation to develop a strategic plan and roadmap for long-term sustainability ("IGSN2040") and to scale operations to the growing volume and diversity of registered objects. The outcome of the IGSN2040 project is a partnership with DataCite (https://blog.datacite.org/igsn-and-datacite-partnership/ and https://blog.datacite.org/igsn-2040-vision-to-execution/)). DataCite will operate the IGSN registration services, leveraging its DOI registration services and business model, while the IGSN e.V. is promoting and supporting a 'community of communities' for standard methods of identifying, citing, and locating physical samples. See:
Lehnert, K., Klump, J., Ramdeen, S., Wyborn, L., & Haak, L. (2021). IGSN 2040 Summary Report: Defining the Future of the IGSN as a Global Persistent Identifier for Material Samples. Zenodo. https://doi.org/10.5281/zenodo.5118289

**Reply by ph-ilten** [reply]
The PYTHIA event generator in high energy particle physics has been around for 40 years, and has been funded from a variety of sources.
- [history of PYTHIA](https://arxiv.org/pdf/1907.09874.pdf)
- [PYTHIA webpage](https://pythia.org/)
- [PYTHIA repo](https://gitlab.com/Pythia8/releases)
